pkg/handler/processor/guesser/type_cyclonedx_test.go

```diff
@@ -0,0 +1,91 @@
+//
+// Copyright 2022 The GUAC Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package guesser
+
+import (
+	"testing"
+
+	"github.com/guacsec/guac/internal/testing/testdata"
+	"github.com/guacsec/guac/pkg/handler/processor"
+)
+
+func Test_cyclonedxTypeGuesser_GuessDocumentType(t *testing.T) {
```

Comment:
there's a test for the generic guesser as well https://github.com/guacsec/guac/blob/main/pkg/handler/processor/guesser/guesser_test.go